docs/categories/index.xml

```diff
@@ -2,13 +2,10 @@
 <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
   <channel>
     <title>Categories on GRIP</title>
-    <link>https://bmeg.github.io/grip/categories/</link>
+    <link>http://localhost:1313/grip/categories/</link>
```

Comment:
The doc build was done with the wrong hostname

Reply:
addressed in newest commit